5_analyze_data/notebooks/KS-test_analysis/visualize_ks_test.r

```diff
@@ -4,7 +4,7 @@ suppressPackageStartupMessages(library(dplyr))
 output_fig_dir <- "figures"
 
 ks_test_fig <- file.path(output_fig_dir, "ks_test_cp_genotype.png")
-actin_distrib_fig <- file.path(output_fig_dir, "top_actin_feature_distrib.png")
+actin_distrib_fig <- file.path(output_fig_dir, "top_er_feature_distrib.png")
```

Comment:
Wouldn't you want to change the name of the variable to `er_distrib_fig`? This is nit picky so it doesn't need to be changed but just noticed the mismatch.

Reply:
I noticed you do change the variable names below this.

Reply:
good point! I will change

Reply:
fixed in a20d3ba918db648a547f92a55520092190e798ab

Going to merge! :rocket: